.github/file-filters.yml

```diff
@@ -16,17 +16,26 @@ yarn_lockfile: &yarn_lockfile
 eslint_config: &eslint_config
   - '.eslint*'
 
-frontend: &frontend
+# `frontend_src` filters on files that are frontend changes excluding
+# changes to the tests/ directory.
+# If you want to filter on *all* frontend files, use the `frontend_all` filter.
+frontend_src: &frontend_src
   - *yarn_lockfile
-  - *frontend_lintable
   - *eslint_config
+  - '!(test)/**/*.[tj]{s,sx}'
+  - '**/tsconfig*.json'
+  - '{package,now,vercel}.json'
   - '**/*.less'
   - 'docs-ui/**'
   - 'static/**'
-  - 'tests/js/**'
   - 'fixtures/search-syntax/**/*'
   - '.github/workflows/frontend.yml'
 
+frontend_all: &frontend_all
+  - *frontend_src
+  - '**/*.[tj]{s,sx}'
+  - 'tests/js/**'
```

Comment:
oh hmmm I guess this gets trickier since the tests just got moved to be next to the source files.  bleh

Reply:
Sorry - the tests filter isn't needed here because the previous line will include them.

But yes, the frontend change is less straightforward.